api/v1/mongodbcommunity_types.go

```diff
@@ -327,8 +327,9 @@ type AutomationConfigOverride struct {
 
 // OverrideProcess contains fields that we can override on the AutomationConfig processes.
 type OverrideProcess struct {
-	Name     string `json:"name"`
-	Disabled bool   `json:"disabled"`
+	Name      string                      `json:"name"`
+	Disabled  bool                        `json:"disabled"`
+	LogRotate *automationconfig.LogRotate `json:"LogRotate,omitempty"`
```

Comment:
Such that we can override this setting per process in case we want a different behaviour per proc